### DISCUSSION TITLE
Having issues in migrating app data in Django

### DISCUSSION THREAD

**Discussion by Shinzo-ninja** [community/community#56571] [Programming Help]
### Body

Having issues in migrating app data in Django
has only one app but still by runing 
`python manage.py migrate`
having error 
django.db.utils.OperationalError: "Table already exists"

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hridesh-net** [comment]
`python manage.py migrate <app_name> zero`
after this run 
`python manage.py migrate
`

**Reply by Shinzo-ninja** [reply]
Thank you

**Comment by queenofcorgis** [comment]
@Shinzo-ninja @hridesh-net GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
